src/traces/pie/index.js

```diff
@@ -373,6 +373,12 @@ pie.plot = function(gd, cdpie) {
                     }
                 }
 
+                function handleClick (evt) {
+                    gd._hoverdata = pt;
+                    gd._hoverdata.trace = slices[0][0].__data__.trace;
```

Comment:
Looks like `pt` already has a reference to `trace`

![image](https://cloud.githubusercontent.com/assets/6675409/11788830/e982502c-a260-11e5-9bb2-e7a50425faf0.png)